mix.exs

```diff
@@ -33,7 +37,11 @@ defmodule Ext.MixProject do
       {:postgrex, ">= 0.0.0-rc"},
       {:plug_cowboy, "~> 2.0"},
       {:logger_file_backend, "~> 0.0.10"},
-      {:mock, "0.3.3", only: :test}
+      {:mock, "0.3.3", only: :test},
+      {:joken, "~> 2.0"},
+      {:ja_serializer, "~> 0.13.0"},
+      {:ecto_enum, "~> 1.0"},
+      {:dataloader, "~> 1.0.0"}
```

Comment:
we really need dataloader?

Reply:
I added this deps to remove warnings during tests